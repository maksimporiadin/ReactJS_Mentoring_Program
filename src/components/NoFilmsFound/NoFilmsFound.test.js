import 'jsdom-global/register';
import React from 'react';
import { NoFilmsFound } from './index';
import { shallow } from 'enzyme';

describe('NoFilmsFound', () => {
    let wrapper;

    it('is renders', () => {
        wrapper = shallow(<NoFilmsFound  />);
        expect(wrapper).toMatchSnapshot();
    });
});