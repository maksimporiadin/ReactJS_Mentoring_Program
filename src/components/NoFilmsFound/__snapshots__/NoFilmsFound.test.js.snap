// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoFilmsFound is renders 1`] = `
<div
  className="noFilmsFound-container"
>
  No Films Found
</div>
`;
